Uplift/Views/GymDetail/GymDetailWeekView.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  GymDetailCalendarView.swift
+//  Uplift
+//
+//  The header calendar used to select which content to display based on day of the week
+//
+//  Created by Phillip OReggio on 10/2/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class GymDetailWeekView: UIView {
+
+    // MARK: - Views
+    private var weekdayCollectionView: UICollectionView!
+
+    // MARK: - Info
+    private var days: [WeekDay] = [.sunday, .monday, .tuesday, .wednesday, .thursday, .friday, .saturday]
```

Comment:
Maybe do `WeekDay.allCases` instead of writing out all values?

Reply:
oh good call